public/content/sv/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,56 @@
+# Välkommen till D
+
+Välkommen till den interaktiva rundtur av *Programmings språket D*.
+
+{{#dmanmobile}}
+
+Denna rundtud ger en överblick av detta __kraftfulla__ och __målande__
+språk vilket kompilerar direkt till __effektiv__, __nativ__ maskin kod.
+
+{{/dmanmobile}}
+
+### Vad är D?
+
+D är höjdpunkten utav __decennier av erfarenhet innom att implementera kompilatorer__
+för många diverse språk och har en stort antal [unika funktioner](http://dlang.org/overview.html):
+
+{{#dmandesktop}}
+
+- _hög nivå_ begrepp för fantastiskt modellerings kraft
+- _hög presterande_, kompilerat språk
+- statisk typning
+- evolutionen av C++ (utan dess misstag)
+- direkt gränssnitt till operativ systemets API:n och hårdvara
+- supersnabb kompilerings tid
+- tillåter minnes-säker programmering (SafeD)
+- _underhållbart_, _enkelt att förstå_ kod
+- kort inlärningskurva (C-likt syntax, likt Java och andra språk)
+- kompatibel med C applikationers binära gränssnitt
+- multi-paradigm (imperative, structured, object oriented, generic, functional programming purity, and even assembly) //TODO: Translate to Swedish
+- inbyggt fel hinder (kontrakt, unitttest)
+
+... och många mer [funktioner](http://dlang.org/overview.html).
+
+{{/dmandesktop}}
+
+### Om rundturen
+
+Varje sektion kommer med ett kod exempel som kan modifieras och användas med för
+att experimentera med Ds språk funktioner.
+Klicka på kör knappen (eller `Ctrl+enter`) för att kompilera och köra koden.
+
+### Bidra
+
+Denna rundtur är [öppen källkod](https://github.com/stonemaster/dlang-tour)
+och vi är glada i 'pull requests' som gör denna rundtur ännu bättre.
+
+## {SourceCode}
+
+```d
+import std.stdio;
+
+void main()
+{
+    writeln("Hello World!");
```

Comment:
> Should those phrases be translated too?

tl;dr: It's your personal choice.

The programming books in German that I read in my youth always started with "Hallo Welt". However after skimming the different Wikipedia pages, I think there's a different cultural attitude if I am not mistake?

https://sv.wikipedia.org/wiki/Hello_World
https://de.wikipedia.org/wiki/Hallo-Welt-Programm

OT: This might be due to the phrase that I heard a couple of times "Swedish people speak better English than the English"